packages/lang/locales/en.json

```diff
@@ -133,6 +133,11 @@
   "dashboard.selected": "Selected",
   "dashboard.masqueradeAction": "Masquerade",
   "dashboard.endMasqueradeAction": "End Masquerade",
+  "dashboard.startMasqueradeModal.title": "Start Masquerading",
+  "dashboard.startMasqueradeModal.message": "Starting masquerade will refresh your shopping cart. Do you want to continue?",
```

Comment:
not sure if "refresh" is the correct word here. When I see that I imagine same products but new information. @bc-sylvialin what do you think?